src/plugins/data_views/common/data_views/data_views.test.ts

```diff
@@ -136,7 +136,7 @@ describe('IndexPatterns', () => {
     SOClientGetDelay = 1000;
     const id = '1';
     setDocsourcePayload(id, {
-      id: 'foo',
+      id,
       version: 'foo',
       attributes: {
```

Comment:
this would need some checking, because, why does the unit test fail, without this change? anyhow, the id should be `1` and not `foo`. I wonder if this is a result of making use of Map instead of Object for the hash?

Reply:
>I wonder if this is a result of making use of Map instead of Object for the hash?

Maybe? I'd like to make sure we understand

Reply:
can be explorered when it's a separate PR, it's weird